setup.cfg

```
@@ -36,12 +36,12 @@ install_requires =
     tornado>=6.1.0
     jupyter_core
     jupyterlab_server~=2.10
-    jupyter_server~=1.4
+    jupyter_server~=1.8
```

Let's not change the dependency for a patch version

```suggestion
    jupyter_server~=1.4
```